addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -584,7 +585,7 @@ def test_discounted_tax(self):
         self.assertTaxTotals(document, {
             'amount_total': 1230.41,
             'amount_untaxed': 1020.41,
-            'display_tax_base': False,
+            'display_tax_base': True,
```

Comment:
Don't you have to repeat those modifications? I'm thinking about  `PurchaseTestTaxTotals`.

Reply:
Nope, the goodness spreads to [Purchase,Sale]TestTaxTotals